automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -28,13 +29,17 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
   val swatTestBucket = "gs://leonardo-swat-test-bucket-do-not-delete"
   val incorrectJupyterExtensionUri = swatTestBucket + "/"
 
-  // Ron and Hermione are on the dev Leo whitelist.
-  val ronAuthToken = AuthToken(LeonardoConfig.Users.ron)
-  val hermioneAuthToken = AuthToken(LeonardoConfig.Users.hermione)
-  val ronEmail = LeonardoConfig.Users.ron.email
+  // Ron and Hermione are on the dev Leo whitelist, and Hermione is a Project Owner
```

Comment:
What whitelist? :)

Reply:
Heh, good point.  I would like to defer those changes please :)

Reply:
+1